services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -95,8 +95,6 @@
 
   private final static String      PLATFORM_WEB_CONTRIBUTORS_GROUP  = "/platform/web-contributors";
```

Comment:
you should also modify canReactOnSpace from social in order to not allow admins to react on redactional space